tests/test_oauth2client.py

```diff
@@ -144,6 +146,23 @@ def get(self, key, default=None):
     return Info(self._headers)
 
 
+@contextlib.contextmanager
+def mock_module_import(module):
+  """Place a dummy object in sys.modules to mock an import test"""
+  parts = module.split('.')
+  for n in range(len(parts)):
```

Comment:
If I say that I prefer "i" for an index and "n" to represent a quantity, am I being too fussy?